NEWS.md

```diff
@@ -27,6 +29,8 @@ shiny 1.4.0.9001
 
 * Added a `'function'` class to `reactive()` and `reactiveVal()` objects. ([#2793](https://github.com/rstudio/shiny/pull/2793))
 
+* Added function `tabPanelBody()` which wraps `tabPanel()` without the title argument.  This function should be used within `tabsetPanel(type = "hidden")` ([#2814](https://github.com/rstudio/shiny/pull/2814))
+
```

Comment:
I would collapse these two bullets into a single bullet that gets the idea across more quickly, something like

* Added a new option (`type = "hidden"`) to `tabsetPanel()`, making it easier to set the active tab via other input controls (e.g., `radioButtons()`) rather than tabs or pills. Use this option in conjunction with `updateTabsetPanel()` and the new `tabsetPanelBody()` function (see `help(tabsetPanel)` for an example and more details).